ssvsigner/tls/tls.go

```diff
@@ -0,0 +1,48 @@
+package tls
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"os"
+)
+
+const (
+	minimumTLSVersion = tls.VersionTLS13
+)
+
+// LoadTLSConfig loads cert/key and optional CA bundle.
+// If requireClientCert is true, the CA bundle is used for client‐auth mutual TLS.
+func LoadTLSConfig(certPath, keyPath, caPath string, requireClientCert bool) (*tls.Config, error) {
```

Comment:
is there a reason not to require all files always? we always pass true to this function